benchmarks/asv.conf.json

```diff
@@ -23,7 +23,7 @@
     //  * No build-time environment variables.
```

Comment:
I know it's no change, but I think the highlighting here is really complaining that "//" comments aren't really allowed in JSON syntax.

https://stackoverflow.com/questions/244777/can-comments-be-used-in-json/4183018#4183018

But the comments there are not encouraging, 
it seems if you want to be 100% "correct" there is no decent way around this.
So if this is useful, and we are currently OK with it, it should probably stay